charts/atlantis/templates/statefulset.yaml

```diff
@@ -387,6 +387,26 @@ spec:
             value: "/var/github-app/key.pem"
           {{- end }}
           {{- end }}
+          {{- if .Values.gitea.user }}
+          - name: ATLANTIS_GITEA_USER
+            value: {{ required "gitea.user is required if gitea configuration is specified." .Values.gitea.user }}
+          - name: ATLANTIS_GITEA_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "atlantis.vcsSecretName" . }}
+                key: gitea_token
+          {{- if .Values.gitea.baseURL }}
```

Comment:
should we aim for gitea.com if not provided?

Reply:
I'd say, let's use the same logic as the others and leave it empty when not provided:

- https://github.com/runatlantis/helm-charts/blob/ad4e15511afe9ee70162ee13487eec98f9976539/charts/atlantis/values.yaml#L57-L62
- https://github.com/runatlantis/helm-charts/blob/ad4e15511afe9ee70162ee13487eec98f9976539/charts/atlantis/values.yaml#L71-L76